blog/2024-06-05.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Getting Started on NEAR Using Windows
+authors: [flmel]
+slug: getting-started-on-windows
+tags: [windows, tutorial, getting-started]
+hide_table_of_contents: true
+---
+In this article, we will cover how to install WSL and setup a NEAR development environment on Windows.
+
+<!-- truncate -->
+
+### WSL
+
+WSL, or Windows Subsystem for Linux, is a compatibility layer for running Linux binary executables natively on Windows. It allows us to run a GNU/Linux environment directly on Windows without the overhead of a traditional virtual machine or dual-boot setup.
+
+#### Enable WSL from `Windows Features`
+
+`Control Panel > Programs > Turn Windows Features on or off`  
+Scrolling down will reveal the option `Windows Subsystem for Linux`. Make sure it is enabled and press OK to confirm (you will be asked to reboot your computer).
+
+![Windows Features WSL Enabled](../website/static/docs/blog/windows-features-wsl-enabled.jpg)
+
+#### Start WSL and Install Ubuntu
+
+Now we will spend some time in either `PowerShell` or [Windows Terminal](https://aka.ms/terminal), which is a modern terminal application that supports various command-line tools and shells.
+
+```
+wsl --install --d Ubuntu
+```
+
+**Read more on WSL:**  
+https://learn.microsoft.com/en-us/windows/wsl/install  
+https://learn.microsoft.com/en-us/windows/wsl/setup/environment  
+https://learn.microsoft.com/en-us/windows/wsl/
+
+### Install Required Packages
```

Comment:
Once the installation is completed you will have Ubuntu installed as any other application on Windows. Open it from the start menu and you will be dropped into the [Ubuntu bash shell](https://ubuntu.com/tutorials/command-line-for-beginners#1-overview)